src/Compiler/Build/WebForms.Compiler.DefaultItems.props

```diff
@@ -11,17 +11,19 @@
   <ItemGroup>
     <WebFormsFiles Include="$(MSBuildProjectDirectory)\**\*.aspx" Exclude="@(__DefaultExcludes)" />
     <WebFormsFiles Include="$(MSBuildProjectDirectory)\**\*.master" Exclude="@(__DefaultExcludes) "/>
+    <WebFormsFiles Include="$(MSBuildProjectDirectory)\**\*.ascx*" Exclude="@(__DefaultExcludes)" />
```

Comment:
that warning you're seeing is probably because of this. change it to `$(MSBuildProjectDirectory)\**\*.ascx`. i.e. remove the asterisk at the end